app/models/round.py

```diff
@@ -1,18 +1,55 @@
+from __future__ import annotations
+
 import inspect
 from dataclasses import dataclass
 
 
+@dataclass
+class ContactDetails:
+    phone: str
+    email_address: str
+    text_phone: str
+
+    @classmethod
+    def from_dict(cls, d: dict):
+        # Filter unknown fields from JSON dictionary
+        return cls(
+            **{
+                k: v
+                for k, v in d.items()
+                if k in inspect.signature(cls).parameters
+            }
+        )
+
+
+@dataclass
+class SupportAvailability:
+    time: str
+    days: str
+
+    @classmethod
+    def from_dict(cls, d: dict):
+        # Filter unknown fields from JSON dictionary
+        return cls(
+            **{
+                k: v
+                for k, v in d.items()
+                if k in inspect.signature(cls).parameters
+            }
+        )
+
+
```

Comment:
Are these needed if we're just using these as dicts? SupportAvailablity has a "closed" attribute too currently (from fund_store) so maybe should either remove these and change the contact_details support_avaialability types to dict (Lines 51/52 below). Or have them explicitly match what's coming from fund_store and instantiate them as ContactDetails/SupportAvailability objects when creating the Round object from_dict()?

Reply:
Will fix in future PR